Failed to run demo-a3-headless after apostrophe update on 3.0.0-aplha.3 version

## To Reproduce

> 1. Go to this repo and download it to your local machine
> 2. Install dependencies
> 3. Start demo with "yarn dev" (if you're using yarn) or "npm run dev"
> 4. You will get OperationalError: ./node_modules/lowlight/lib/core.js
Module not found: Error: Can't resolve 'highlight.js/lib/core' in '/Users/nikolalakovic/Documents/sitestorm/demo-a3-headless/node_modules/lowlight/lib'
>5. After manual installation of 'lowlight' library (yarn add lowlight) everything is fine

--- a/package.json
+++ b/package.json
@@ -20,6 +20,7 @@
     "babel-loader": "^8.1.0",
     "css-loader": "^4.3.0",
     "extract-css-chunks-webpack-plugin": "^4.8.0",
+    "lowlight": "^1.19.0",
     "sass": "^1.26.11",
     "sass-loader": "^10.0.2",
     "style-loader": "^1.2.1",

